util/test_timeline.py

```diff
@@ -370,24 +347,36 @@ def run_file(args, timelist):
         'branch': 1,
         'timeline_stopped': True
     }
-    file_started = False
+    started = False
 
     with args.file as file:
+        # If searching for encounters, divert and find start/end first3
+        if args.search_fights:
+            encounter_sets = e_find.find_fights_in_file(file)
+            # If all we want to do is list encounters, stop here and give to the user.
+            if args.search_fights == -1:
+                return [f'{i + 1}. {" ".join(e_info)}' for i, e_info in enumerate(encounter_sets)]
+            elif args.search_fights > len(encounter_sets) or args.search_fights < -1:
+                raise Exception('Selected fight index not in selected ACT log.')
+
+        start_time, end_time = e_find.choose_fight_times(args, encounter_sets)
```

Comment:
Like you do in `make_timeline`, I think you need to assign encounter_sets to be [] outside of the above conditional, or it might be unassigned here.